php/php.editor/src/org/netbeans/modules/php/editor/verification/UnusableTypesHintError.java

```diff
@@ -430,18 +430,15 @@ private void checkDuplicateType(List<Expression> types) {
                 }
                 if (checkedTypes.contains(name)) {
                     createDuplicateTypeError(type, typeName);
-                    return;
```

Comment:
I've added it for a little performance (because the next error is shown if the error is fixed) but I don't think this fix has a big problem. So, it's OK.

Reply:
Right, original code with early exit finishes faster.
If there is no error then all types will be checked anyway.